elide-core/src/main/java/com/yahoo/elide/parsers/state/CollectionTerminalState.java

```diff
@@ -28,11 +27,7 @@
 import lombok.ToString;
 import org.apache.commons.lang3.tuple.Pair;
 
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
Our style is not to have wildcards in the code (although no style check currently catches this).